scripts/deploy/troubleshooting/get-all-logs

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+function usage() {
+  echo "Usage: $0 <namespace>"
+  echo
+  echo "This script saves logs from all pods within the specified namespace into local files and then creates a tar.gz archive of these logs."
+  echo
+  echo "Arguments:"
+  echo "  namespace   - The namespace from which to retrieve and archive pod logs."
+  echo
+  echo "Example: $0 mynamespace"
+  exit 1
+}
+
+if [ "$#" -ne 1 ]; then
+  usage
+fi
+
+NAMESPACE=$1
+
+mkdir -p "$NAMESPACE-logs"
+if [ $? -ne 0 ]; then
+  echo "Failed to create directory for logs."
+  exit 1
+fi
+
+# Get a list of all pod names in the namespace
+PODS=$(kubectl -n $NAMESPACE get pods -o jsonpath="{.items[*].metadata.name}")
+
+if [ -z "$PODS" ]; then
+  echo "No Pods found in Namespace $NAMESPACE."
+  rm -rf "$NAMESPACE-logs"
+  exit 1
+fi
+
+# Loop through all pod names and fetch logs, saving them to individual files
+for POD in $PODS; do
+  kubectl -n $NAMESPACE logs $POD > "$NAMESPACE-logs/$POD.log"
```

Comment:
One issue that the docker compose version of this script faced was that some deployments were long running and would return months (years!?) of logs which was way way too much.  By default it would return 24 hrs worth of logs but would allow you to [specify your own range](https://docs.communityhealthtoolkit.org/apps/guides/debugging/sharing-4x-logs/#calling-the-script).

We should add that feature here!

Here's a mock code update.  We'd need to accept a new input instead hard coding it to `24h`. as well, we need to use the new `tmp` directory suggested above:

```bash
kubectl -n $NAMESPACE logs $POD  --since=24h > "/tmp/cht-eks-logs/$POD.log"
```

Reply:
Handled.